404.html

```diff
@@ -1,20 +1,15 @@
 <!DOCTYPE html>
 <html lang="en">
 
-<head>
-  <meta charset="utf-8" />
-  <script>
-    sessionStorage.redirect = location.href;
-  </script>
-  <meta http-equiv="refresh" content="0;URL='/'">
-  </meta>
-</head>
+  <head>
+    <meta charset="utf-8"/>
+    <script>
+      sessionStorage.redirect = location.href;
+    </script>
+    <meta http-equiv="refresh" content="0;URL='/'">
+    <link rel="stylesheet" href="/index.css">
+  </head>
 
-<body>
-  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
This is required to make GitHub pages recognise it as a real 404.html (has to be over 512 bytes). Might not still be a thing.. I haven't checked recently.

Reply:
Maybe add an encouraging bon mot instead? 

"No codez for you lol 😂" or similar?